driver/SCHEMA_VERSION

```diff
@@ -1 +1 @@
-2.7.0
```

Comment:
We shouldn't bump the schema version.

https://github.com/falcosecurity/libs/blob/master/proposals/20220203-versioning-schema-amendment.md#drivers-artifacts
> - *patch* increases either when `API_VERSION`’s patch or `SCHEMA_VERSION`’s patch number has been increased **or when any other code changes have been introduced (for example, the support for a new kernel)**

This is exactly the case when `API_VERSION` is unchanged and `SCHEMA_VERSION` is unchanged, but still in the next driver release, we have to bump the driver patch number.

Does it make sense?

Reply:
yep, makes sense thank you for pointing this out!